src/lib/openapi/spec/token-string-list-schema.ts

```diff
@@ -0,0 +1,23 @@
+import { edgeTokenSchema } from './edge-token-schema';
+import { FromSchema } from 'json-schema-to-ts';
+
+export const tokenStringListSchema = {
+    $id: '#/components/schemas/tokenStringListSchema',
+    type: 'object',
+    additionalProperties: false,
+    required: ['tokens'],
+    properties: {
+        tokens: {
+            description: 'Tokens used to authenticate against Unleash',
```

Comment:
How are the tokens presented? Just as raw strings? This might be clearer with the example, but are admin, front-end, client tokens, etc all in the same list? (Unleash itself knows what kind of tokens they are, right?)